### DISCUSSION TITLE
Dark theme isnt working on app opening screen

### DISCUSSION THREAD

**Discussion by Divivats** [ReVanced/revanced-discussions#1105] [Question]
Since the last 2 patches, the app opening screen isn't following dark mode, its always in light theme. The rest of the app is working in dark mode but the opening screen where youtube logo appears is in light theme.

https://github.com/user-attachments/assets/a01476a2-053a-4b8f-a210-6e8f95280b94

**Comment by oSumAtrIX** [comment]
Same occurs on the unpatched app

**Reply by Divivats** [reply]
Ok, thanks 👍

**Comment by LisoUseInAIKyrios** [comment]
The Theme patch changes this behavior, and the dark splash screen is changed to the dark theme color (default dark color is flat black).
